CHANGELOG

```diff
@@ -1,4 +1,7 @@
 --- CHANGELOG ---
+--- PyFMI-2.10.0 ---
+    * Changed default behavior for Radau5ODE to switch to analytical and sparse Jacobians, dependent on problem size.
```

Comment:
Perhaps we should mention anything about when this is switching, reading this immediately asks the question "when does it switch"?